supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -182,10 +182,10 @@ Admonitions should draw the reader’s attention to certain information. Keep ad
 
 Valid admonition types:
 
-- NOTE
-- IMPORTANT
-- WARNING
-- TIP
+NOTE:: Additional information that adds something to the content, but if you breeze past it, nothing will break.
+IMPORTANT:: Considerations that you should not ignore.
+WARNING:: Ignoring this information can cause system damage or a support-related issue.
+TIP:: Additional information that can help make things easier or more productive for you.
```

Comment:
This suggestion is to change the 'you' to 'users' because this is advice for the content creators.
Also suggesting:

- to change 'things' to 'processes'
- to remove the 'help' because it's fluffy:  #minimalism :-) 

TIP:: Additional information that can make processes easier or more productive for users.

Reply:
I edited this based on comments in chat.